llm/mcp_servers/services/resume_loader_service.py

```
@@ -0,0 +1,71 @@
+            for file in files:
+                if file.lower().endswith('.txt'):
+                    resume_path = os.path.join(root, file)
+                    applicant_name = file[0:len(file) - 4]
+                    applicant_name = applicant_name.replace('_', ' ').replace('-', ' ')
+                    return resume_path, applicant_name
+        return None
+        
+
+    async def get_highlighted_sections(self):
+        resume_setctions_content = await self.read_file(f'{self.ADDITIONAl_FILE_PATH_PREFIX}/resume_sections.txt')
+        if resume_setctions_content == None:
```

Ruff (0.12.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)